src/Database/schema.sql

```diff
@@ -35,6 +35,18 @@ CREATE TABLE labels (
   FOREIGN KEY (book_id) REFERENCES books(id) ON DELETE CASCADE
 );
 
+CREATE TABLE authors (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  first_name VARCHAR(100),
+  last_name VARCHAR(100),
```

Comment:
Please add a foreign key to reference the games table (game_id) &
hint: You can check out how I created the labels table.